src/Common/Commands.Common/MetricHelper.cs

```diff
@@ -207,9 +207,11 @@ public void FlushMetric()
         /// <returns></returns>
         public static string GenerateSha256HashString(string originInput)
```

Comment:
@shahabhijeet It seems that Win10AnniversaryEdition does not properly enforce FIPS-mode. See https://gist.github.com/pburkholder/01214219f89c794a4740c7a59d1846ff#file-findings-md